src/components/Match/Vision/VisionFilter.jsx

```diff
@@ -19,30 +19,30 @@ const data = [
   },
 ];
 
-export default class VisionFilter extends React.PureComponent {
-  handleCheckGenerator(player, type) {
-    const parent = this.props.parent;
-
-    return event => parent.setPlayer(player, type, event.target.checked);
-  }
-
-  handleCheckGeneratorTeam(index) {
-    const parent = this.props.parent;
-
-    return event => parent.setTeam(index === 0 ? 'radiant' : 'dire', event.target.checked);
-  }
```

Comment:
My concern is that splitting off the function like this causes the developer to have to jump around the code to find the values that are getting passed.  In the new version, the variables playerNumber, row, and  event are all in the same contiguous block.

If the function needed to be reused, then we'd have to split it off, but that currently isn't the case.